exoctk/exoctk_app/app_exoctk.py

```diff
@@ -672,6 +673,209 @@ def save_fortney_result():
+                inputs[key] = val
+            else:
+                print('This parameter was unfit : {}'.format(key))
+                error_message = 'One of the scaling parameters was out of range: {}.'.format(key)
+                break
+        
+        # Map to nearest model key
+        temp_range = np.arange(400, 2700, 100)
+        grav_range = np.array([5, 10, 20, 50])
+        sort_temp = (np.abs(inputs['temperature'] - temp_range)).argmin()
+        sort_grav = (np.abs(inputs['gravity'] - grav_range)).argmin()
+        model_temp = temp_range[sort_temp]
+        input_args['model_temperature'] = '0{}'.format(model_temp)[-4:]
+        model_grav = grav_range[sort_grav]
+        input_args['model_gravity'] = '0{}'.format(model_grav)[-2:]
+        print(model_temp, model_grav)
+
+        # Check the model parameters
+        str_temp_range = ['0{}'.format(elem)[-4:] for elem in temp_range]
+        model_space = [('condensation', ['local', 'rainout']), 
+                       ('model_temperature', str_temp_range),
+                       ('model_gravity', ['05', '10', '20', '50']),
+                       ('metallicity', ['+0.0', '+1.0', '+1.7', '+2.0', '+2.3']),
+                       ('c_o', ['0.35', '0.56', '0.70', '1.00']),
+                       ('haze', ['0001', '0010', '0150', '1100']),
+                       ('cloud', ['0.00', '0.06', '0.20','1.00'])]
+        
+        model_key = ''
+        for tup in model_space:
+            key, space = tup
+            if input_args[key] in space:
+                inputs[key] = input_args[key]
+                model_key += '{}_'.format(inputs[key])
+            else:
+                error_message = 'One of the model parameters was out of range.'
+                break
+        model_key = model_key[:-1]
+    
+
+        # Define constants
+        boltzmann = 1.380658E-16 # gm*cm^2/s^2 * Kelvin
+        permitivity = 1.6726E-24 * 2.3 #g  cgs  Hydrogen + Helium Atmosphere
+        optical_depth = 0.56 
+        r_sun = 69580000000 # cm
+        r_jupiter = 6991100000 # cm
+ 
+        closest_match = {'model_key': model_key, 'model_gravity': model_grav,
+                         'model_temperature': model_temp}
+        generic_db = '/user/jfowler/exoctk_work/generic/generic_grid_db.hdf5'
+        with h5py.File(generic_db, 'r') as f:
+            # Can't use the final NaN value
+            model_wv = f['/wavelength'][...][:-1]
+            model_spectra = f['/spectra/{}'.format(model_key)][...][:-1]
+            
+        radius_ratio = np.sqrt(model_spectra) * inputs['r_planet']/inputs['r_star']
+        r_star = inputs['r_star'] * r_sun
+        r_planet = inputs['r_planet'] * r_jupiter
+        model_grav = model_grav * 1e2
+        inputs['gravity'] = inputs['gravity'] * 1e2
+        
+        # Start with baseline based on model parameters
+        scale_height = (boltzmann * model_temp) / (permitivity * model_grav)
+        print(scale_height)
```

Comment:
Are these print statements for testing or output for the user? If for testing, nuke 'em. If for the user, perhaps add some string formatting like "Scale height: {}" or something?